webAO/master.js

```diff
@@ -1,151 +1,138 @@
 import FingerprintJS from '@fingerprintjs/fingerprintjs';
-
-import { unescapeChat, safeTags } from './encoding.js';
-
-const myStorage = window.localStorage;
```

Comment:
did you delete the localStorage

Reply:
Yes, I deleted `myStorage` because you didn't reference the variable anywhere.